tuna/connection.py

```diff
@@ -241,36 +263,43 @@ def ssh_connect(self, abort=None):
     self.ssh = paramiko.SSHClient()
     self.ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
     for ssh_idx in range(NUM_SSH_RETRIES):
-      if abort is not None and chk_abort_file(self.id, self.logger):
-        self.logger.warning('Machine %s aborted ssh connection', self.id)
+      if abort is not None and chk_abort_file(id, self.logger):
+        self.logger.warning('Machine %s aborted ssh connection', id)
```

Comment:
id not valid here, see previous comment

Reply:
it is initialised in __init__ funciton. now we can use id, along with self key reference.